drivers/hmis/app/graphql/types/forms/form_definition.rb

```diff
@@ -10,10 +10,7 @@ module Types
   class Forms::FormDefinition < Types::BaseObject
     description 'FormDefinition'
     field :id, ID, null: false
-    field :version, Int, null: false
```

Comment:
I'm missing something, why do we no longer need version of the form definition?  Because they are stored alongside the data?

Reply:
we never used this in graphql. version is only used to determine which is the most "current" form to use when rendering an assessment